### DISCUSSION TITLE
Output specific variables from MFront to CalculiX frd output file

### DISCUSSION THREAD

**Discussion by JAWilliams123** [thelfer/tfel#116] [Q&A] [answered]
Hello Everyone,

Is there a way to output say the damage d for each finite element in CalculiX when using the Unilateral Mazars damage model? 

Many thanks,
JWilliams

**Comment by thelfer** [comment]
@JAWilliams123 To be honest, I have no clue. You'd better ask this question on a forum dedicated to `CalculiX`. However, do not hesitate to give any meaningful solution here.

**Reply by JAWilliams123** [reply]
@thelfer, many thanks! I will follow through with a query on the CalculiX discussion group and will leave this discussion open until I share the feedback. 

Best wishes,
JWilliams

**Comment by JAWilliams123** [comment, marked as answer]
Hello Everyone,

So to get StateVariables printed in the output file the command lines below do the trick:
*EL FILE
SDV

Many thanks to @rafal.brzegowy on the CalculiX discussion page.

Best wishes,
JWilliams

**Reply by thelfer** [reply]
@JAWilliams123 Thanks for the feed-back.
